web/modules/custom/default_content_config/content/node/44a761c7-41e3-4657-99bc-7248846cd073.yml

```diff
@@ -57,11 +56,6 @@ default:
           field_guide_link:
             -
               entity: 62fdaf72-5449-4312-a82e-b87b9b406f36
-  field_kicker_link:
```

Comment:
Did you mean to delete this or is this another case of those silly circular references? I think I merged that code into develop if you want to update this branch.

Reply:
fixed the exports